### PR TITLE
feat: Add test to pomodoro-clock-tests

### PR DESCRIPTION
**Pull request by RShalman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `master` branch of freeCodeCamp/testable-projects-fcc
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] Your changes have been tested either locally or using a newly created CDN based on your fork's testable-projects-fcc/build/bundle.js file

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
  
- [ ] Closes currently open issue: Closes #XXX

#### Description

This PR adds new test to `pomodoro-clock-tests` that checks if `time-left` stays unchanged if breakMin or breakPlus are clicked. Because if that happen, and you press `start` - clock starts with break time instead of session time. That's the case when a developer doesn't check `timeType`. You can check how it works by changing `if (this.state.timerType === timerType)` in the example project to `if (this.state.timerType === !timerType)` (simply set ! before `timerType`) and then running tests.

### CODE REVIEW

File: src/project-tests/pomodoro-clock-tests.js [hunk 1]
```diff
@@ -291,6 +291,29 @@ export default function createPomodoroClockTests() {
         );
       });
 
+      it(`When I click the element with the id of "break-decrement" and then with the
+        id of "break-increment", both times element with the id "time-left" stays unchanged 
+        (time stays equal to the value of the element with the id "session-length")`, async function() {
```
Comment (tbushman) [inline src/project-tests/pomodoro-clock-tests.js]: Thank you for your contribution. Please change the wording to 'When I click the elements with the id of "break-decrement" and "break-increment" at the same time, the time indicated by the element with the id, "time-left", remains unchanged and equal to the value of the element with the id, "session-length".'

Comment (RShalman) [reply src/project-tests/pomodoro-clock-tests.js]: @tbushman done!)

**Comment by tbushman** [comment]
Thanks! Since you are working on the tests for the Pomodoro project, maybe you have some ideas on the existing issues with the Pomodoro tests. Look for the help wanted tag.   
LGTM!

**Comment by RandellDawson** [comment]
@tbushman Does the curriculum need to be updated also with any changes made on this PR?

**Comment by tbushman** [comment]
@RandellDawson No. I'd say not. :) Take care.

**Comment by tbushman** [comment]
@RandellDawson # edit: it does add a user story.

**Comment by RandellDawson** [comment]
@tbushman We need a way to make sure changes in User Stories get reflected in the curriculum.

Also, by adding a new user story, will this cause older projects to break (not pass the tests)?

**Comment by tbushman** [comment]
@RandellDawson @RShalman 
I'm considering un-doing the merge. The added test checks that increment / decrement buttons, clicked at the same time, cancel each other out / have no effect. The test is not critical to assessment nor understanding.  
Mainly I'm hoping that you, @RShalman, and anyone else, will feel welcome to work on any of the long-standing Pomodoro clock issues. I have marked many with the help-wanted tag.

**Pull request merged**